### ISSUE TITLE
ctc_decode(greedy=Fasle) return wrong log probability of each decoded sequence

### ISSUE THREAD

**Issue by theiron97** [keras-team/keras#10813]
I use K.ctc_decode(y_pred=y_pred, input_length = seq_len, greedy=False, beam_width=100, top_paths=10)[1]. Then I get result : 
![keras](https://user-images.githubusercontent.com/15943606/43451663-a81288a4-94df-11e8-8390-7c9f7b1303da.png)
It seem wrong becase log(p) = 0.07480443 ??

**Comment by yanboliang** [comment]
@theiron97 I think you hit https://github.com/tensorflow/tensorflow/issues/6034 , this should be a TensorFlow bug, as it doesn't consider normalization term inside ```tf.nn.ctc_beam_search_decoder```. There is a fix at  https://github.com/tensorflow/tensorflow/pull/21187 . For your case, you can consider switch to the greedy version, it should work well.

**Comment by theiron97** [comment]
Thank for your answer. But I want to find top paths of decoder so I can't use greedy version.
